src/gui/properties/peersadditiondlg.h

```diff
@@ -32,29 +32,28 @@
 #define PEERADDITION_H
 
 #include <QDialog>
-#include "ui_peer.h"
 
-class QHostAddress;
+#include "core/bittorrent/peerinfo.h"
+#include "ui_peersadditiondlg.h"
 
 namespace BitTorrent
 {
     struct PeerAddress;
```

Comment:
I suppose this forward declaration doesn't work and you needed to include peerinfo,h, correct? If yes, remove this then. Otherwise remove the include.